### PR TITLE
onefetch 2.12.0

### PR DESCRIPTION
**Pull request by o2sh**
Created by https://github.com/mislav/bump-homebrew-formula-action

### CODE REVIEW

**Comment by o2sh** [comment]
The [test](https://github.com/Homebrew/homebrew-core/blob/739e7e88208524c78a447da6b68dc13ba90b13d2/Formula/onefetch.rb#L33) fails because it does not account for the presence of ANSI escaped codes in the output which was Ok before we changed our coloring library.

The regex should probably be changed to smth like:

```ruby 
assert_match(/Language.*Ruby/, shell_output("#{bin}/onefetch").chomp)
```

**Comment by chenrui333** [comment]
@o2sh mind if updating the test? it would trigger a rerun. thanks!

File: Formula/onefetch.rb [hunk 1]
```diff
@@ -30,6 +30,6 @@ def install
     system "git", "config", "user.name", "BrewTestBot"
     system "git", "config", "user.email", "BrewTestBot@test.com"
     system "echo \"puts 'Hello, world'\" > main.rb && git add main.rb && git commit -m \"First commit\""
-    assert_match(/Language:.*Ruby/, shell_output("#{bin}/onefetch").chomp)
+    assert_match(/Ruby (100.0 %)/, shell_output("#{bin}/onefetch").chomp)
```
Comment (cho-m) [inline Formula/onefetch.rb:33]: Current failure is due to parentheses being seen as regex capture group.

Can just match on string literal. 
```suggestion
    assert_match "Ruby (100.0 %)", shell_output("#{bin}/onefetch").chomp
```

Comment (o2sh) [reply Formula/onefetch.rb]: Thanks a lot for the help @cho-m

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**